fractions/javaee/jaxrs/src/main/java/org/wildfly/swarm/jaxrs/JAXRSMessages.java

```diff
@@ -0,0 +1,42 @@
+/*
+ * JBoss, Home of Professional Open Source.
+ *
+ * Copyright 2017 Red Hat, Inc., and individual contributors
+ * as indicated by the @author tags.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.wildfly.swarm.jaxrs;
+
+import org.jboss.logging.BasicLogger;
+import org.jboss.logging.Logger;
+import org.jboss.logging.annotations.LogMessage;
+import org.jboss.logging.annotations.Message;
+import org.jboss.logging.annotations.MessageLogger;
+
+/**
+ * @author Martin Kouba
+ */
+@MessageLogger(projectCode = "WFSWARM", length = 4)
```

Comment:
I've actually been going for WFSSOMETHING for each new @MessageLogger, to avoid having to deal with offsets across the entire project.

We don't need to stick to just on WFSWARM.  WFSJAXRS works for me.